.travis.yml

```diff
@@ -9,18 +9,68 @@ branches:
   only:
     - master
 
-php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
-    - 7.0
-
-env:
-    - WP_VERSION=nightly WP_MULTISITE=0
-    - WP_VERSION=nightly WP_MULTISITE=1
-    - WP_VERSION=4.4 WP_MULTISITE=0
-    - WP_VERSION=4.4 WP_MULTISITE=0
+matrix:
```

Comment:
no way to make this less verbose?

Reply:
Only by dropping 5.3 support. It's a bug in Travis, as 5.3 requires the `precise` distro. https://github.com/travis-ci/travis-yaml/issues/118